src/components/command-bar/commands/search/unified-search/components/unified-hit/index.tsx

```diff
@@ -0,0 +1,132 @@
+// Components
+import CardLink from 'components/card-link'
+import Text from 'components/text'
+// Helpers
+import { buildUrlPath, renderHighlightArrayHtml } from './helpers'
+// Types
+import type { Hit } from 'instantsearch.js'
+// Styles
+import s from './unified-hit.module.css'
+
+/**
+ * TODO: this is a placeholder component for a "hit" card component.
+ * It can likely be ignored for now.
+ *
+ * We'll replace this with something properly usable in a future pass
+ * to the "All" tab work for unified search.
+ */
+export function UnifiedHit({ hit }: { hit: Hit }) {
```

Comment:
Generally: Intent is to ignore the `unified-hit` work for now.

This component is the "ugly first draft" part of this PR. It's chonky and lengthy and can mostly be ignored I think - intent is to just get _something_ rendering that shows everything else is working, or to put it another way, intent is to refine the "hit" cards in a later PR rather than within this PR.